### DISCUSSION TITLE
How To get Selected Page DomComponents

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#4808] [Q&A] [answered]
I am Looking for Some DomComponents to Perform Some Actions but its in different Page How to get DomComponents on Selected Page

```
for example: editor.Pages.getSelected().DomComponents.getWrapper()
```
Any Options Like This

**Comment by artf** [comment, marked as answer]
You can use [`Pages.getSelected()`](https://grapesjs.com/docs/api/pages.html#getselected) and [`page.getMainComponent()`](https://grapesjs.com/docs/api/page.html#getmaincomponent) API
```js
const selectedPage = editor.Pages.getSelected();
const rootComponent = selectedPage.getMainComponent();
console.log(rootComponent.toHTML());
```
